### PR TITLE
[ADD] odoo-shippable: Install python 3.6 and python 3.6-dev

### PR DESCRIPTION
**Pull request by JesusZapata**
Fix https://github.com/Vauxoo/docker-odoo-image/issues/228

### CODE REVIEW

**Comment by JesusZapata** [comment]
@moylop260 
This PR install python 3.6 and install virtualenv for all version of python except 2.7

File: odoo-shippable/scripts/build-image.sh [hunk 1]
```diff
@@ -20,6 +20,8 @@ PYTHON_PPA_REPO="deb http://ppa.launchpad.net/fkrull/deadsnakes/ubuntu trusty ma
 PYTHON_PPA_KEY="http://keyserver.ubuntu.com:11371/pks/lookup?op=get&search=0x5BB92C09DB82666C"
 VIM_PPA_REPO="deb http://ppa.launchpad.net/pkg-vim/vim-daily/ubuntu trusty main"
 VIM_PPA_KEY="http://keyserver.ubuntu.com:11371/pks/lookup?op=get&search=0xA7266A2DD31525A0"
+PY36_PPA_REPO="deb http://ppa.launchpad.net/jonathonf/python-3.6/ubuntu trusty main"
```
Comment (moylop260) [inline odoo-shippable/scripts/build-image.sh]: I'm not sure but I think that the current PPA REPO has python3 and is not required add other one

File: odoo-shippable/scripts/build-image.sh [hunk 2]
```diff
@@ -71,6 +75,13 @@ apt-get install ${DPKG_DEPENDS} ${PIP_DPKG_BUILD_DEPENDS}
 # Install node dependencies
 npm install ${NPM_OPTS} ${NPM_DEPENDS}
 
+# Install virtualenv for each version of python
+for version in '3.3' '3.4' '3.5'
```
Comment (moylop260) [inline odoo-shippable/scripts/build-image.sh]: What about 2.7 and 3.6?

Comment (JesusZapata) [inline odoo-shippable/scripts/build-image.sh]: @moylop260 
I try to run `python3.6 -m pip install virtualenv`
image

File: odoo-shippable/scripts/build-image.sh (hunk 1 above)
Comment (JesusZapata) [inline odoo-shippable/scripts/build-image.sh]: @moylop260 
You're right

File: odoo-shippable/scripts/build-image.sh [hunk 3]
```diff
@@ -71,6 +71,13 @@ apt-get install ${DPKG_DEPENDS} ${PIP_DPKG_BUILD_DEPENDS}
 # Install node dependencies
 npm install ${NPM_OPTS} ${NPM_DEPENDS}
 
+# Install virtualenv for each version of python
+for version in '2.7' '3.3' '3.4' '3.5'
```
Comment (moylop260) [inline odoo-shippable/scripts/build-image.sh]: 3.6?

Comment (JesusZapata) [inline odoo-shippable/scripts/build-image.sh]: @moylop260 
I try to run `python3.6 -m pip install virtualenv` and show this error

image

Comment (moylop260) [inline odoo-shippable/scripts/build-image.sh]: so fix it

**Comment by JesusZapata** [comment]
@moylop260 
Now all versions of python have installed virtualenv
Travis is in red for an isolated case

File: odoo-shippable/scripts/build-image.sh [hunk 4]
```diff
@@ -113,7 +127,9 @@ git_clone_copy "${PYLINT_REPO}" "master" "conf/.jslintrc" "${REPO_REQUIREMENTS}/
 ln -sf ${REPO_REQUIREMENTS}/linit_hook/git/* /usr/share/git-core/templates/hooks/
 
 # Creating virtual environments for python and node js
-virtualenv --system-site-packages ${REPO_REQUIREMENTS}/virtualenv/python2.7
+export VIRTUALENVWRAPPER_PYTHON=/usr/bin/python2.7
+echo "VIRTUALENVWRAPPER_PYTHON=/usr/bin/python2.7"
```
Comment (moylop260) [inline odoo-shippable/scripts/build-image.sh]: This "echo" has a effect?

Comment (JesusZapata) [inline odoo-shippable/scripts/build-image.sh]: @moylop260 
Sorry, I forget to add `>> /etc/bash.bashrc`

**Pull request merged**